tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/CiamIntegrationTests.cs

```diff
@@ -26,11 +26,9 @@ public class CiamIntegrationTests
         private const string _ciamRedirectUri = "http://localhost";
 
         [TestMethod]
-        [DataRow("https://{0}.ciamlogin.com/", 0)] //https://tenantName.ciamlogin.com/
-        [DataRow("https://{0}.ciamlogin.com/{1}.onmicrosoft.com", 1)] //https://tenantName.ciamlogin.com/tenantName.onmicrosoft.com
-        [DataRow("https://{0}.ciamlogin.com/{1}", 2)] //https://tenantName.ciamlogin.com/tenantId
-        public async Task ROPC_Ciam_Async(string authorityFormat, int authorityVersion)
+        public async Task ROPC_Ciam_Async()
```

Comment:
Some test naming suggestions:
- `Async` is not needed in test names (I suppressed that rule), it doesn't add useful info.
- `Ciam` can be removed to keep the name short. The class name tells us that all of these tests are CIAM.

I suggest following these test naming guidelines:
[Naming your tests](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests)

> The name of your test should consist of three parts:
> The name of the method being tested.
> The scenario under which it's being tested.
> The expected behavior when the scenario is invoked.

So naming could be something like:
- `ROPC_StandardDomains_ReturnsValidTokens` or `ROPC_StandardDomains_CompletesSuccessfully`
- `ClientCredential_WithClientSecret_ReturnsValidTokens`
- `OBO_CustomDomain_ReturnsValidTokens'

Reply:
Removing Ciam makes it hard to tell which tests are failing when looking at the build.

Reply:
Well if you click on the test name, it shows the class name in the details/stack trace, but I'll agree that it's not obvious at first glance in the failed tests list.